libs/api/domains/financial-statements-inao/src/lib/client/dataverseClient.ts

```diff
@@ -0,0 +1,70 @@
+import { createEnhancedFetch } from '@island.is/clients/middlewares'
+import { Inject, Injectable } from '@nestjs/common'
+
+import { ClientType } from '../models/clientType.model'
+import { Election } from '../models/election.model'
+import {
+  DataverseClientConfig,
+  DATAVERSE_CLIENT_CONFIG,
+} from './dataverseClientConfig'
+
+@Injectable()
+export class DataverseClient {
```

Comment:
Wondering if we should have this as a separate client library in `libs/clients`

Reply:
I discussed this with Eiki few days ago, and since this client won't be used by any other app/lib there is no need to make this into a separate lib. But we are trying to make an openApi doc for the Dataverse API, but haven't been successful yet, but if that change, we will probably move this into a client lib.

Reply:
Alright :)